HorticraftPricer.ahk

```diff
@@ -437,12 +437,14 @@ valuedictionary[RAW_Jewel_Implicit_Cluster] := Jewel_Implicit_Cluster
 
 ; You can modify your keybind here, see AHK's documentation on keybinds here: https://www.autohotkey.com/docs/Hotkeys.htm
 ; By default we are set to '!w' or, 'Alt W'
-!w Up::
+!c Up::
```

Comment:
The README file was wrong and has been updated, Alt+W was correct, I changed it to that a bit ago.